bin/dropboxignore.sh

```diff
@@ -545,6 +545,7 @@ Usage: "$PROGRAM_NAME" <command> <path> [-v 0-2] [-p pattern]
 
   Commands:
 
+    gui			generate - update - ignore in 1 shortcut command
```

Comment:
Well, `gui` abbreviation could be misunderstood because is inextricably linked with the concept of Graphical User Interface. I think that we should find a different action name. What do you think? My proposal is to adopt `combo` but feel free to propose a different name.

```suggestion
    combo               Generate, update and ignore at once.
```